src/internal/api/packages/find.go

```diff
@@ -40,9 +41,14 @@ func findPackage(pattern *regexp.Regexp, w http.ResponseWriter, setDir func() (s
 	targetDir, err := setDir()
 	if err != nil {
 		message.ErrorWebf(err, w, "Error getting directory")
+		return
 	}
 
-	// Intentionally ignore errors
-	files, _ := utils.RecursiveFileList(targetDir, pattern)
+	files, err := utils.RecursiveFileList(targetDir, pattern)
+	if err != nil || len(files) == 0 {
```

Comment:
Leaving a comment here just to denote the fact that we talked in person about eventually splitting this error return out into an bigger obj that can capture more detail on what specifically went wrong. Eventually...